app/grants/management/commands/estimate_clr.py

```diff
@@ -49,24 +49,11 @@ def handle(self, *args, **options):
 
         if active_clr_rounds:
             for clr_round in active_clr_rounds:
-                print(f"CALCULATING CLR estimates for ROUND: {clr_round.round_num}")
-                predict_clr(
+                process_predict_clr(
```

Comment:
fwiw; this doesnt actually delegate the tasks to the queues.  it just runs it inline.